homeassistant/components/switch/doorbird.py

```diff
@@ -0,0 +1,103 @@
+"""Support for powering relays in a DoorBird video doorbell."""
+import datetime
+import logging
+import voluptuous as vol
+
+from homeassistant.components.doorbird import DOMAIN
+from homeassistant.components.switch import SwitchDevice, PLATFORM_SCHEMA
+from homeassistant.const import CONF_SWITCHES
+import homeassistant.helpers.config_validation as cv
+
+DEPENDENCIES = ['doorbird']
+
+_LOGGER = logging.getLogger(__name__)
+
+SWITCHES = {
+    "open_door": {
+        "name": "Open Door",
+        "icon": {
+            True: "lock-open",
+            False: "lock"
+        },
+        "time": datetime.timedelta(seconds=3)
+    },
+    "light_on": {
+        "name": "Light On",
+        "icon": {
+            True: "lightbulb-on",
+            False: "lightbulb"
+        },
+        "time": datetime.timedelta(minutes=5)
+    }
+}
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_SWITCHES, default=[]):
```

Comment:
If one was to be the default, I'd say `light_on` would be a better choice. It turns on the IR array for night vision, which all DoorBird devices include. The `open_door` switch is only useful if you have an electronic door strike installed.